GooglePlayInstant/Editor/GooglePlayServices/CommandLine.cs

```diff
@@ -677,6 +677,14 @@ public static string QuotePath(string path)
         {
             return string.Format("\"{0}\"", path);
         }
+        
+        /// <summary>
+        /// Returns the specified argument in double quotes if it contains a space.
+        /// </summary>
+        public static string QuoteIfNecessary(string path)
```

Comment:
It doesn't seem like this is called, so should it be deleted?

Reply:
Yes, thanks for catching that!